### PR TITLE
Nisha | Stage | Provide support of master export in Giddh

### PR DESCRIPTION
**Pull request by nisha-walkover**
https://app.clickup.com/t/86cvqx99a

### CODE REVIEW

**Review by dvCodeWorld — APPROVED** [review]

File: apps/web-giddh/src/app/shared/header/components/master-export-option/master-export-option.component.ts [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  /* This will hold local JSON data */
+  public localeData: any = {};
+  /* This will hold common JSON data */
+  public commonLocaleData: any = {};
+  constructor(
+    private formBuilder: FormBuilder
+  ) { }
+
+  /**
+   * This will use for export form
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/shared/header/components/master-export-option/master-export-option.component.ts]: correct this documentation

File: apps/web-giddh/src/app/shared/header/components/new-manage-groups-accounts/manage-groups-accounts.component.ts [hunk 2]
```diff
@@ -49,6 +49,8 @@ export class ManageGroupsAccountsComponent implements OnInit, OnDestroy, AfterVi
     private hasUnsavedChanges: boolean = false;
     /** True if confirmation is open on search groups/accounts keyup event */
     private isPageLeaveConfirmationOpen: boolean = false;
+    /** shows active group unique name */
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/shared/header/components/new-manage-groups-accounts/manage-groups-accounts.component.ts]:
```suggestion
    /** Hold active group unique name */
```

**Review by dilpreetsinghofficial — CHANGES_REQUESTED** [review]

**Review by dilpreetsinghofficial — APPROVED** [review]

File: apps/web-giddh/src/app/shared/header/components/master-export-option/master-export-option.component.ts (hunk 1 above)
Comment (nisha-walkover) [reply apps/web-giddh/src/app/shared/header/components/master-export-option/master-export-option.component.ts]: done

File: apps/web-giddh/src/app/shared/header/components/new-manage-groups-accounts/manage-groups-accounts.component.ts (hunk 2 above)
Comment (nisha-walkover) [reply apps/web-giddh/src/app/shared/header/components/new-manage-groups-accounts/manage-groups-accounts.component.ts]: done

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**